frontend/src/pages/home.tsx

```diff
@@ -245,7 +245,12 @@ const Home: FC = () => {
                       <h3 className="mb-8 font-display text-xl font-light md:mb-11 md:text-3xl">
                         {t('tabs.plan.linksTitle')}
                       </h3>
-                      <List disablePadding>
+                      <List disablePadding className='ml-4'
+                      sx={{
+                        '.MuiButtonBase-root.MuiListItemButton-root': {
+                          paddingLeft: '0px',
+                        },
+                      }}>
```

Comment:
Removing the left padding looks weird since it's an anchor button.

![image](https://github.com/DTS-STN/senior-journey/assets/114004123/bde92bf8-fd17-4cf0-b41b-cb87e44ae8b8)

Use `className="pl-0"` on `ListItemButton` component instead of using `sx` prop.

Reply:
The design might have to change to have no padding... maybe the link should be on the item header and removing the right nav icon.

Reply:
swapped it to className,